app/web/service/resources.ts

```diff
@@ -20,3 +20,8 @@ export async function getLanguages() {
   const languages = await client.resources.getLanguages(new Empty());
   return languages.toObject();
 }
+
+export async function getBadges() {
```

Comment:
this was already implemented by the backend??

Reply:
No, that's from #4246 that this one builds on!

Reply:
gocha!